extensions/wikia/Search/SearchApiController.class.php

```diff
@@ -129,96 +138,108 @@ public function getCombined() {
 		if ( $this->request->getVal( 'lang' ) ) {
 			throw new InvalidParameterApiException( 'lang' );
 		}
-		$minArticleQuality = $this->request->getVal(self::MIN_ARTICLE_QUALITY_PARAM_NAME, self::DEFAULT_MIN_ARTICLE_QUALITY);
+		$minArticleQuality =
+			$this->request->getVal( self::MIN_ARTICLE_QUALITY_PARAM_NAME, self::DEFAULT_MIN_ARTICLE_QUALITY );
 
 		$hubs = $this->request->getArray( 'hubs' );
 		foreach ( $hubs as $hub ) {
-			if ( !isset( $this->allowedHubs[ $hub ] ) ) {
+			if ( !isset( $this->allowedHubs[$hub] ) ) {
 				$hub = htmlentities( $hub, ENT_QUOTES );
 				throw new InvalidParameterApiException( 'hubs (' . $hub . ')' );
 			}
 		}
 
 		$query = $this->request->getVal( 'query' );
-		$langs = $this->request->getArray( 'langs', ['en'] );
+		$langs = $this->request->getArray( 'langs', [ 'en' ] );
 		$namespaces = $this->request->getArray( 'namespaces', [ NS_MAIN ] );
 		$limit = $this->request->getVal( 'limit', null );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "limit" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)